meta-xt-prod-cockpit-rcar-app/recipes-cluster/cluster-bin/cluster-wrapper_git.bb

```diff
@@ -14,7 +14,7 @@ SRC_URI = " \
    git://git@gitpct.epam.com/epmd-aepr/vlib;protocol=ssh;branch=${BRANCH} \
 "
 
-SRCREV = "d5d52c4baf782aaef287efa3450046ac4f22dacf"
+SRCREV = "046a0cbe6f628e098527943edadafd2a47879143"
```

Comment:
This seems like not very good idea. Value provided here will never be used, I suspect.
`SRCREV` mentioned here will be always overridden by the `SRCREV_pn-cluster-wrapper` from the `local.conf` (i.e. from the yaml-file). So, if you want set SRCREV from the yaml, it will be better to remove this line and add comment, that "SRCREV is set through the local.conf and should be provided in the yaml".

But from other side, overriding is standard behavior of the yocto... so, let is be as is.